crashlytics/src/cpp/common/crashlytics.cc

```diff
@@ -21,6 +21,7 @@
 #include "app/src/assert.h"
 #include "app/src/cleanup_notifier.h"
 #include "app/src/include/firebase/app.h"
+#include "app/src/include/firebase/internal/mutex.h"
```

Comment:
Same here

Reply:
remove the original one to keep the alphabetical sequence